_data/changelogs/component-date-picker.yml

```diff
@@ -2,6 +2,14 @@ title: Date picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Enhanced `aria-disabled` behaviors to match `disabled` behaviors.
+    summaryAdditional: Now, the input element and calendar icon remain discoverable to screen readers but cannot be interacted with.
```

Comment:
**Suggestion (non-blocking):** Ok sorry to double back to this but having second thoughts about "cannot be interacted with. Considering something like:

```suggestion
    summaryAdditional: Now, the input element and calendar icon remain discoverable to screen readers but prevent the user from selecting a date.
```

CC: @amyleadem cus she's good at these... 😁

Reply:
Resolved in [63ee9fc](https://github.com/uswds/uswds-site/pull/2794/commits/63ee9fccd2f039087be6d17a0450ffc4caa19d46). Made a small update to what you recommended @mahoneycm so let me know what you think!